generators/chipyard/src/main/scala/config/RocketConfigs.scala

```diff
@@ -205,6 +205,24 @@ class SmallSPIFlashRocketConfig extends Config(
   new freechips.rocketchip.subsystem.WithCoherentBusTopology ++
   new freechips.rocketchip.system.BaseConfig)
 
+class SimAXIRocketConfig extends Config(
+  new chipyard.iobinders.WithUARTAdapter ++
+  new chipyard.iobinders.WithTieOffInterrupts ++
+  new chipyard.iobinders.WithSimAXIMem ++                       // drive the master AXI4 memory with a SimAXIMem
```

Comment:
Is the difference between this and simDRAM that this uses a 1-cycle magic memory and simDRAM is realistic? If so, it might be worth noting that.

Reply:
"realistic"